src/jazayeri_lab_to_nwb/watters/watters_convert_session.py

```diff
@@ -31,107 +58,134 @@ def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str,
-    processed_source_data.update(dict(LF=dict(file_path=str(lfp_files[0]))))
+    logging.info('    LFP data')
+    lfp_file = _get_single_file(probe_data_dir, suffix='.lf.bin')
+    raw_source_data.update(dict(LF=dict(file_path=lfp_file)))
+    processed_source_data.update(dict(LF=dict(file_path=lfp_file)))
     raw_conversion_options.update(dict(LF=dict(stub_test=stub_test)))
-    processed_conversion_options.update(dict(LF=dict(stub_test=stub_test, write_electrical_series=False)))
+    processed_conversion_options.update(
+        dict(LF=dict(stub_test=stub_test, write_electrical_series=False)))
 
     # Add Sorting
+    logging.info('    Spike sorted data')
     processed_source_data.update(
         dict(
             SortingNP=dict(
-                folder_path=str(data_dir_path / "spike_sorting_raw" / "np"),
+                folder_path=str(data_dir / "spike_sorting_raw" / "np"),
                 keep_good_only=False,
             )
         )
     )
-    processed_conversion_options.update(dict(SortingNP=dict(stub_test=stub_test, write_as="processing")))
+    processed_conversion_options.update(
+        dict(SortingNP=dict(stub_test=stub_test, write_as="processing")))
 
     # Add Behavior
+    logging.info('Adding behavior')
+    behavior_path = str(data_dir / "data_open_source" / "behavior")
     processed_source_data.update(
-        dict(EyePosition=dict(folder_path=str(data_dir_path / "data_open_source" / "behavior")))
+        dict(EyePosition=dict(folder_path=behavior_path))
     )
     processed_conversion_options.update(dict(EyePosition=dict()))
 
-    processed_source_data.update(dict(PupilSize=dict(folder_path=str(data_dir_path / "data_open_source" / "behavior"))))
+    processed_source_data.update(
+        dict(PupilSize=dict(folder_path=behavior_path)))
     processed_conversion_options.update(dict(PupilSize=dict()))
 
     # Add Trials
-    processed_source_data.update(dict(Trials=dict(folder_path=str(data_dir_path / "data_open_source"))))
+    logging.info('Adding task data')
+    processed_source_data.update(
+        dict(Trials=dict(folder_path=str(data_dir / "data_open_source"))))
     processed_conversion_options.update(dict(Trials=dict()))
 
     processed_converter = WattersNWBConverter(
-        source_data=processed_source_data, sync_dir=str(data_dir_path / "sync_pulses")
+        source_data=processed_source_data,
+        sync_dir=str(data_dir / "sync_pulses")
     )
 
     # Add datetime to conversion
-    metadata = processed_converter.get_metadata()  # use processed b/c it has everything
+    metadata = processed_converter.get_metadata()
     metadata["NWBFile"]["session_id"] = session_id
 
     # Subject name
-    if "monkey0" in str(data_dir_path):
-        metadata["Subject"]["subject_id"] = "Perle"
-    elif "monkey1" in str(data_dir_path):
-        metadata["Subject"]["subject_id"] = "Elgar"
+    if "monkey0" in str(data_dir):
+        metadata["Subject"]["subject_id"] = "P"
```

Comment:
FYI this will be a value that shows up in the final filename on DANDI - I highly recommend making it the full name or some other easily identifiable/communicable string